TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Queries/CreateInductionQuery.cs

```diff
@@ -0,0 +1,11 @@
+namespace TeachingRecordSystem.Core.Dqt.Queries;
+
+public record CreateInductionQuery : ICrmQuery<Guid>
+{
+    public required Guid? PersonId { get; init; }
+    public required DateTime? StartDate { get; init; }
+    public required DateTime? CompletionDate { get; init; }
+    public required dfeta_InductionStatus? InductionStatus { get; init; }
```

Comment:
Does this need to be nullable?
```suggestion
    public required dfeta_InductionStatus InductionStatus { get; init; }
```